openfpga_flow/tasks/fpga_verilog/synthesizable_verilog/config/task.conf

```diff
@@ -0,0 +1,39 @@
+# = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
```

Comment:
I like the test case and we should add it to our CI. 
Can you update our CI script by adding this test case? I think fpga_verilog test is a proper place to add. See t[he example](https://github.com/lnis-uofu/OpenFPGA/blob/45e8baf98f4454335155824f403887bd861ac280/.github/workflows/fpga_verilog_reg_test.sh#L75)